pyproject.toml

```
@@ -24,7 +24,8 @@ dependencies = [
     "tqdm>=4.50.2",
     "sparse>=0.13.0",
     "numba>=0.56.4",
-    "pydantic>=1.10, <2.0.0",
+    "pydantic>=2.5.3, <3.0.0",
```

Removing version cap. And changing version to match that in the requirements.txt
Trying to avoid artificially cappy when we don't need to. 
We can always add back in if this becomes an issue again.
```suggestion
    "pydantic>=2.0.0",
```

---

done

---

Sorted